src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -100,7 +95,7 @@ public void periodic()
         }
 
         // Update odometry
-        _poseEstimator.update(_gyroInputs.yawPosition, getModulePositions());
+        _poseEstimator.update(_gyro._inputs.yawPosition, getModulePositions());
```

Comment:
We'll need to update this to use the new functions we created in the `Gyro` subsystem:

```suggestion
        _poseEstimator.update(_gyro.getYawPosition(), getModulePositions());
```